FWCore/src/PodioDataSvc.cpp

```diff
@@ -38,6 +39,13 @@ StatusCode PodioDataSvc::clearStore()    {
   return StatusCode::SUCCESS ;
 }
 
+void PodioDataSvc::setCollectionIDs(podio::CollectionIDTable* collectionIds) {
+  if (m_collectionIDs != nullptr) {
```

Comment:
As discussed: I agree but some changes to PODIO interfaces are necessary to make that a neat solution.